keras_cv/layers/preprocessing/resizing.py

```diff
@@ -59,6 +59,8 @@ class Resizing(BaseImageAugmentationLayer):
             largest possible resize of the image (of size `(height, width)`) that
             matches the target aspect ratio. By default
             (`pad_to_aspect_ratio=False`), aspect ratio may not be preserved.
+        pad_only: If True, resize the images without aspect ratio distortion only using
```

Comment:
Maybe this API change should be a separate PR?

Reply:
Removed - yeah good point its probably not the right move